packages/address-consts/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "2.0.1",
   "license": "MIT",
   "description": "Constants and types relating to `@shopify/address`",
-  "main": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
+  "main": "index.js",
```

Comment:
Is outputting an `index.js` in the root a defaults from sewing-kit next? I personally like when all the build output is in one folder that way it's easier to undo/clear caches. Probably worth a discussion with the team though.

Reply:
I can look into this; I think it's the default, not sure if it's configureable (based on `PackageEntryOptions` in SK-next). FWIW, I added a `yarn clean` script to `package.json` that clears away any built files/entrypoints.

Reply:
After seeing the use case for multiple entries into a package (eg. `web-workers`), this sounds like a reasonable tradeoff. 

```tsx
   pkg.entry({root: './src/index'});
   pkg.entry({name: 'babel', root: './src/babel-plugin'});
   pkg.entry({name: 'webpack', root: './src/webpack-parts'});
   pkg.entry({name: 'worker', root: './src/worker'});
```

This whole automation of a package entry is really neat!

Reply:
Yeah, I don't love littering the root of a package with all its entry files, but unfortunately there's not really any other way to do multi-entry packages consistently.